x/intertx/types/v1/features/msg_register_account.feature

```diff
@@ -0,0 +1,53 @@
+Feature: MsgRegisterAccount
```

Comment:
would be nice to have a test case for amino sign bytes.

Reply:
added for MsgRegisterAccount https://github.com/regen-network/regen-ledger/pull/1477/commits/e0746403460f9765fb36eea3233d54a4c98b4710

dont think its possible for MsgSubmitTx though, since it contains `protobuf.Any`

Reply:
The `MsgSubmitTx`'s `GetSignBytes` method panicking because, we need to register `intertx` codec with every other modules `RegisterLegacyAmino` method. We had a similar issue with SDK's governance module `MsgSubmitProposal` [13196](https://github.com/cosmos/cosmos-sdk/pull/13196).

Reply:
oof, that just makes it impossible to use via amino right? that doesn't seem very good...

Reply:
Not sure this will work. We can import all modules `RegisterLegacyAminoCodec()` method and register with `intertx` module codec.